src/fmu/ensemble/etc.py

```diff
@@ -9,6 +9,8 @@
 
 # pylint: disable=protected-access
 
+warnings.filterwarnings(action="always", category=DeprecationWarning, module=r".*etc")
```

Comment:
Is the ".*etc" a bit too aggressive?

Reply:
Fixed. Trying to catch multiple import use-cases.